docs/index.rst

```diff
@@ -14,9 +14,12 @@ Contents:
    readme
    installation
    usage
+   load
```

Comment:
I would combine these two sections into one called parsing. Right now there's too much duplication as the only difference between `load` and `loads` is the input is a stream vs a string.